components/ChipTextField/examples/MDCChipTextField.m

```diff
@@ -29,144 +37,254 @@ @implementation MDCChipTextField
 - (instancetype)initWithFrame:(CGRect)frame {
   self = [super initWithFrame:frame];
   if (self) {
-    _chipsView = [[UIView alloc] initWithFrame:CGRectZero];
-    _chipsView.translatesAutoresizingMaskIntoConstraints = NO;
-    //_chipsView.backgroundColor = [UIColor yellowColor];
-    _chipsView.clipsToBounds = YES;
-    self.leftView = _chipsView;
+    _chipViews = [[NSMutableArray alloc] init];
+    _chipsContainerHorizontalOffsetForTextEditing = 0.0f;
+    _chipsContainerHorizontalWidthForTextEditing = 0.0f;
+    [self setupChipsContainerView];
 
     [self addTarget:self
-                  action:@selector(chipTextFieldTextDidChange)
-        forControlEvents:UIControlEventEditingChanged];
-
-    _chips = [NSMutableArray array];
+             action:@selector(chipTextFieldTextDidChange)
+   forControlEvents:UIControlEventEditingChanged];
   }
   return self;
 }
 
+- (void)setupChipsContainerView {
+  MDCChipTextFieldScrollView *chipsContainerView = [[MDCChipTextFieldScrollView alloc] initWithFrame:CGRectZero];
+  chipsContainerView.translatesAutoresizingMaskIntoConstraints = NO;
+//  chipsContainerView.chipSpacing = kChipsSpacing;
```

Comment:
Should we remove this?

Reply:
Addressed in efcb81c4eed97ffe93b9378271fa5c91d227a63a.